src/include/btree_inline.h

```diff
@@ -2269,6 +2269,57 @@ __wt_btcur_bounds_early_exit(
     return (0);
 }
 
+/*
+ * __wt_btcur_skip_inmem_reconcile_page --
+ *     Return if the cursor is pointing to an in-memory page reconciliation image with deleted
```

Comment:
What is "in-memory page reconciliation image"? Do you mean the page is being reconciled?

Reply:
I think you mean page has been reconciled.